test/Common.FunctionalTests/OutOfProcess/AspNetCorePortTests.cs

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Sockets;
+using System.Threading.Tasks;
+using Xunit;
+using Microsoft.AspNetCore.Server.IIS.FunctionalTests.Utilities;
+using Microsoft.AspNetCore.Server.IntegrationTesting;
+using Microsoft.AspNetCore.Testing.xunit;
+
+namespace Microsoft.AspNetCore.Server.IISIntegration.FunctionalTests
+{
+    [Collection(PublishedSitesCollection.Name)]
+    public class AspNetCorePortTests : IISFunctionalTestBase
+    {
+        private static readonly Random _random = new Random();
+
+        private readonly PublishedSitesFixture _fixture;
+
+        public AspNetCorePortTests(PublishedSitesFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        public static TestMatrix TestVariants
```

Comment:
Would this return inprocess variants?

Reply:
Actually `OutOfProcess\HelloWorldTest.cs` doesn't set this explicitly either.  Do you think it should?

Reply:
I don't have a strong opinion on this. I think it's better to be explicit but your test would fail if ran with inproc anyway.